### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by twonirwana**

### CODE REVIEW

**Comment by twonirwana** [comment]
@Swiip Happy new year, can i ask you to take a quick look at the french translation? I don't speak french and used deepl to translate it.

File: bot/src/main/resources/discord/appDirectory/description_fr.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Ce bot sert à lancer des dés dans discord.
+Le bot doit être configuré avec une commande slash dans un canal et fournit ensuite un message avec des boutons.
+En cliquant sur un bouton, le bot lancera l'expression de dé stockée pour le bouton, affichera le résultat et déplacera le message avec les boutons vers le bas du canal.
+Si le message est en attente, il sera seulement copié et non déplacé.
+Cela permet à l'utilisateur de lancer les dés sans avoir à taper de commandes et améliore ainsi la convivialité, en particulier pour les utilisateurs d'écrans tactiles.
+Le bot prend en charge les fils de discussion Discord (le message du bouton doit être ajouté après la création du fil de discussion), les forums, le texte en voix, et il est possible d'envoyer la réponse dans un canal différent.
```
Comment (Swiip) [inline bot/src/main/resources/discord/appDirectory/description_fr.md:6]: le texte en voix -> Text in Voice (keep english here, we're used to and translating it feels wrong)

File: bot/src/main/resources/discord/appDirectory/description_fr.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+* créer des commandes d'application (qui sont nécessaires pour contrôler le bot)
+* envoyer des messages et Envoyer des messages dans les fils (pour les boutons de dés et les résultats)
+* intégrer des liens (affichage des réponses structurées)
+* joindre des fichiers (ajouter des images avec le résultat du dé)
+* voir les anciens messagess (pour détecter si le message d'un bouton a été épinglé)
+
+# Démarrage rapide
+
+Tapez `/quickstart système' et le bot vous proposera une liste de jeux de dés prêts à jouer. Sélectionnez simplement un système dans la liste ou continuez à taper pour chercher et filtrer dans la liste.
```
Comment (Swiip) [inline bot/src/main/resources/discord/appDirectory/description_fr.md:19]: `/quickstart système' -> `/quickstart system' (no translate here)

File: bot/src/main/resources/discord/appDirectory/description_fr.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Beaucoup d'autres systèmes peuvent être cartographiés avec ce bot, veuillez visiter : https://github.com/twonirwana/DiscordDiceBot
```
Comment (Swiip) [inline bot/src/main/resources/discord/appDirectory/description_fr.md:21]: cartographiés -> mappé (kind of mixed french / english here)

**Comment by Swiip** [comment]
Hey 👋  (and happy new year)

The translation was really good already, I just proposed some adjustments

File: bot/src/main/resources/discord/appDirectory/description_fr.md (hunk 2 above)
Comment (twonirwana) [reply bot/src/main/resources/discord/appDirectory/description_fr.md:19]: I translated the command options to in botMessages_fr.properties , so /quickstart système is currently correct

Comment (Swiip) [reply bot/src/main/resources/discord/appDirectory/description_fr.md:19]: Ah, I see sorry

**Comment by twonirwana** [comment]
> Hey 👋 (and happy new year)
> 
> The translation was really good already, I just proposed some adjustments

thanks a lot

**Pull request merged**